### ISSUE TITLE
feature request: add paste verb

### ISSUE THREAD

**Issue by aborruso** [johnkerl/miller#335]
Miller is like awk, sed, cut, join, and sort for name-indexed data, but not paste.

To stack data "horizontally" is a typical operation for those involved in processing data, and for this reason I'm asking to add it.

@johnkerl it's only an idea, I know you have many things to do, but I think that a bombastic tool as Miller should have also this feature.

The goal is to have something like `mlr --csv paste ./files*.csv >output.csv`. And if there are input files with different number of records, the output (for a CSV) will have the number of records that has the file with the highest number of records (or if they don't have the same number of records, you have an error). 

Thank you

**Comment by aborruso** [comment]
Only to add that in recent weeks I have discovered another great tool, which does it: https://github.com/jqnatividad/qsv

It's its cat (concatenate) command https://github.com/jqnatividad/qsv/blob/master/src/cmd/cat.rs#L7

**Comment by janxkoci** [comment]
I thought about this surprising omission some time ago, but I thing it's not as much needed as other verbs because the system `paste` generally works well even for csv/tsv files. It may be useful for more fancy stuff, like pasting a subset of columns, or merging e.g. json.

**Comment by aborruso** [comment]
> because the system `paste` generally works well even for csv/tsv files

True, but not all formats are rectangular ;)

**Comment by janxkoci** [comment]
@aborruso yes, it's why I mentioned json 😉 Just I never really work with json, so my data are generally easy to `paste` 😁
